docs/api/zh/core/EventDispatcher.html

```diff
@@ -20,19 +20,15 @@ <h2>代码示例</h2>
 		<code>
 		// 为自定义对象添加事件
 
-		const Car = function () {
+		const Car extends EventDispatcher () {
```

Comment:
It should be just:
```js
const Car extends EventDispatcher {
```

Reply:
> It should be:
> 
> ```js
> class Car extends EventDispatcher {
> ```

TBH, I coded it and after second just copy your code, made this mistake.
Thanks your reviewed, anyway, I changed it.